microsoft-365/security/office-365-security/skip-filtering-phishing-simulations-sec-ops-mailboxes.md

```diff
@@ -144,6 +144,8 @@ The SecOps mailbox entries that you configured are displayed on the **SecOps mai
    > You can specify up to 10 entries for each field.
    >
    > There must be a match on at least one **Domain** and one **Sending IP**, but no association between values is maintained.
+   > 
+   > Important! If your MX record does not point to Office 365, the IP address that is present in the Authentication-results header must match the IP added to the rule. This may require you to configure Enhanced Filtering for Connectors in order for the correct IP to be detected.
```

Comment:
```suggestion
   > It is important to note that if your MX record does not point to Office 365, the IP address that is present in the Authentication-results header must match the IP added to the rule. This may require you to configure Enhanced Filtering for Connectors in order for the correct IP to be detected.
```